test.js

```diff
@@ -109,3 +90,9 @@ it('should error if cannot get repository', function() {
     getPkgRepo({});
   });
 });
+
+it('should parse github enterprise http url', function() {
+  var url = 'http://github.mycompany.dev/user/myRepo';
```

Comment:
Can you assign the parsed result to a local var? also can you test 
```js
this.project = slug[1];
this.committish = slug[2];
```
otherwise I'll add them myself. Thanks!